en/docs/concepts/provisioning-framework.md

```diff
@@ -0,0 +1,37 @@
+
+# Identity provisioning framework 
+
+The identity provisioning framework is the main component that handles user / role provisioning in WSO2 Identity Server. The WSO2 Identity Provisioning framework can be separated into three main components:
+
+- **Inbound provisioning** : used by the external applications to provision users to WSO2 Identity Server. 
+- **Outbound provisioning** : responsible for provisioning users from WSO2 Identity Server to external applications.
+- **Userstore management** : used to persist users within the system
+
+<img name='sso-diagram' src='../../../assets/img/concepts/provisioning-framework.png' class='img-zoomable'/>
+
+## Inbound provisioning requests
+
+The inbound provisioning component focuses on how to provision users to WSO2 Identity Server (WSO2 IS). By default, WSO2 IS supports inbound provisioning via a Simple Object Access Protocol (SOAP) based API as well as the System for Cross-domain Identity Management (SCIM) API. Both APIs support HTTP Basic Authentication. Inbound provisioning requests can come in the form of SCIM or SOAP.
```

Comment:
Don't "Inbound provisioning requests can come in the form of SCIM or SOAP." and "By default, WSO2 IS supports inbound provisioning via a Simple Object Access Protocol (SOAP) based API as well as the System for Cross-domain Identity Management (SCIM) API." mean the same thing? 

Shall we merge these into one sentence so that it's less confusing? WDYT?